### DISCUSSION TITLE
Does the Laravel query builder's `orderBy` function support multiple orders in a single function call?

### DISCUSSION THREAD

**Discussion by kaozaza2** [laravel/framework#48668] [Q&A]
like this:

```php
$query->orderBy([
    'COLUMN1' => 'ASC',
    'COLUMN2' => 'DESC',
]);
```

**Comment by shaedrich** [comment]
Not according [to the docs](https://laravel.com/docs/queries#ordering). `$column` has to be either [`\Closure|\Illuminate\Database\Eloquent\Builder|\Illuminate\Database\Query\Builder|\Illuminate\Database\Query\Expression|string`](https://github.com/laravel/framework/blob/10.x/src/Illuminate/Database/Query/Builder.php#L2306):

> To sort by multiple columns, you may simply invoke orderBy as many times as necessary:
```php
$users = DB::table('users')
                ->orderBy('name', 'desc')
                ->orderBy('email', 'asc')
                ->get();
```

However, alternatively you might try [`orderByRaw()`](https://laravel.com/docs/queries#orderbyraw)
